specs/dwritecore/DWriteCore.md

```diff
@@ -1050,9 +1230,185 @@ int __cdecl wmain(int argc, WCHAR** argv)
 }
 ```
 
-# API Details
+### Rendering Color Glyph Runs
+
+In most cases, the simplest way to render glyph runs with color is to use either Direct2D's or
+DirectWrite's `DrawGlyphRunWithColor` method.
 
-## DWRITE_FACTORY_TYPE Enumeration
+The following example shows a lower-level approach, in which the client calls
+`TranslateColorGlyphRun` to translate the base glyph run into a sequence of color glyph runs, each
```

Comment:
Just so I understand what the Translate is actually doing - if you have a color font and create a default glyph run, you can render it in monochrome, but secretly, it stores color information internally so that if the client wishes to render in color, this translation will use that secret color information to create a sequence of glyph runs with color information?

Reply:
Yes, that's the general idea (except the "secretly" part 😂).